CsvParser/csvParser.go

```diff
@@ -129,3 +138,32 @@ func makeJsonObjFile(arr []incident.Incident) *os.File {
 
 	return f
 }
+
+// // Helper functions to store incidents in according directory
+func addIncidentToJSON(inc incident.IncidentData, jsonFileName string) {
```

Comment:
Let's remove this since I think it will be unused :,(

Reply:
Removed :)